lib/diagnostic.js

```diff
@@ -6,20 +6,23 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (words) {
+  let normalize = words.map(words.replace(/\W+/,'').toUpperCase());
+  return normalize;
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (words) {
+  words.forEach(normalizeWords(word)) {
+  }
+  return
 };
 
 // 3. Write a function that returns the count of words in a string.
 // Provid the *option* to count unique words instead of total words.
-const wordCount = function (/* */) {
-  /* your response here */
+const wordCount = function (words) {
+  return words.length;
 };
 
```

Comment:
This requires that words must be an array, correct? You probably need to call .split somewhere in here.